src/main/resources/application.properties

```diff
@@ -0,0 +1,15 @@
+#Tomcat
+server.servlet.contextPath=/crm-test
+
+#Log
+logging.level.com.agilemonkeys.test.crm = DEBUG
+logging.level.org.springframework.core = INFO
+
+#Spring Security
```

Comment:
I know that this is a test project but as a general rule is better to use environment variables instead of hardcoded security passwords in property files.

Reply:
i also think this. In real environment the properties will be store in environment variables.
The client oauth credential and the token information will be store in DB. If the server is down, all token are secure.